marketplacetests/tests/test_marketplace_in_app_not_you_login_feature.py

```diff
@@ -6,40 +6,41 @@
 from fxapom.fxapom import FxATestAccount
 
 from marketplacetests.firefox_accounts.app import FirefoxAccounts
-from marketplacetests.payment.app import Payment
-from marketplacetests.in_app_payments.in_app import InAppPayment
+from marketplacetests.payment.app import InAppPayment
+from marketplacetests.in_app_payments.in_app import InAppPaymentTester
 from marketplacetests.marketplace_gaia_test import MarketplaceGaiaTestCase
 
 
 class TestNotYouLinkInAppPayment(MarketplaceGaiaTestCase):
 
-    def test_not_you_link_in_app_payment(self):
+    test_data = {
+        'app_name': 'Testing In-App-Payments',
+        'app_title': 'In-App-Payments',
+        'product': 'test 0.99 USD'}
 
-        self.app_name = 'Testing In-App-Payments'
-        app_title = 'In-App-Payments'
+    def test_not_you_link_in_app_payment(self):
```

Comment:
It's not your change, but this test method name doesn't make any sense to me. Could you make it clearer what we're testing here?